decision-records/005-repository-organisation-for-v5.md

```diff
@@ -0,0 +1,120 @@
+# Decision Record Title
+
+Update our repository organisation to prepare v5
```

Comment:
This could also mean moving the repository itself to a new GitHub organisation. To avoid confusion, could we change it to something like:

```suggestion
Update our repository structure to prepare v5
```